articles/azure-functions/functions-app-settings.md

```diff
@@ -433,7 +433,7 @@ The file path to the function app code and configuration in an event-driven scal
 |---|------------|
 |WEBSITE_CONTENTSHARE|`functionapp091999e2`|
 
-Only used when deploying to a Windows or Linux Premium plan or to a Windows Consumption plan. Not supported for Linux Consumption plans or Windows or Linux Dedicated plans. Changing or removing this setting may cause your function app to not start. To learn more, see [this troubleshooting article](functions-recover-storage-account.md#storage-account-application-settings-were-deleted).
+Only used when deploying to a Windows or Linux Premium plan or to a Windows Consumption plan. Not supported for Linux Consumption plans or Windows or Linux Dedicated plans. When changing the setting please ensure the value is lower cased. Changing or removing this setting may cause your function app to not start. To learn more, see [this troubleshooting article](functions-recover-storage-account.md#storage-account-application-settings-were-deleted).
```

Comment:
Would this be more correct, or does a share name never include capital letters?
 ```suggestion
Only used when deploying to a Windows or Linux Premium plan or to a Windows Consumption plan. Not supported for Linux Consumption plans or Windows or Linux Dedicated plans. `WEBSITE_CONTENTSHARE` is case sensitive. Changing or removing this setting may cause your function app to not start. To learn more, see [this troubleshooting article](functions-recover-storage-account.md#storage-account-application-settings-were-deleted).
```

Reply:
@ggailey777 , the setting needs to be lower-cased or else the functions runtime won't be able to start correctly.

Reply:
Thanks @jackbatzner. One more question: in what scenario where you changing the `WEBSITE_CONTENTSHARE` setting?

Reply:
We were setting the value on a first deploy of an Azure function to get it created properly via ARM Templates @ggailey777 . From there we wouldn't set the value anymore.

Reply:
OK, duh this is actually a files storage share naming requirement. Sorry I was so slow on this one. Thanks again for the help! #sign-off